pipelines/release-images-cf-deployment/tasks/build.yml

```diff
@@ -0,0 +1,22 @@
+---
+platform: linux
+image_resource:
+  type: docker-image
+  source:
+    repository: thulioassis/fissile-cf-deployment-ci
```

Comment:
This repo is temporary and will be replaced by the image built with the Dockerfile?

Reply:
Yes. I'll do it.